### PR TITLE
Fix wrong division name on "Manage teams" page

### LINKED ISSUE #99: Wrong Division name is displayed on Manage teams page
1. Create tournament with division with non default name
2. Go to manage teams
3. Add any team to tournament into created division

Division name does not match selected

- [x] #113 - Find a view which displays "Manage teams" page

- [x]  #110  - Track how the data is transmitted to the form

- [x]  #150 - Update UT

### LINKED ISSUE #150: Update UT
Update corresponding UT which is responsible for showing list of teams on view
Parent #99

### PR DESCRIPTION
**Pull request by oleggerus**
### Summary

Changed TeamNameViewModel for showing right Division name on "Manage teams" page.
closes #150

### Checklist

- #113  - Find a view which displays "Manage teams" page

-  #110 - Track how the data is transmitted to the form

-  #150 - Update UT

#### Design
nothing

#### Perfromance
nothing

#### Unit Tests
nothing

### CODE REVIEW

File: src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/Teams/TeamNameViewModel.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ public static TeamNameViewModel Map(TeamTournamentDto team)
             {
                 Id = team.TeamId,
                 Name = team.TeamName,
-                DivisionName = $"Division {team.DivisionId}"
+                DivisionName = team.DivisionName
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/Teams/TeamNameViewModel.cs]: You should not have removed checklist 😄 

- Please update corresponding UT

Comment (oleggerus) [reply src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/Teams/TeamNameViewModel.cs]: UT it is a Unit Test , as I understand?

Comment (sdiachen) [reply src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/Teams/TeamNameViewModel.cs]: yes)

File: src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs [hunk 2]
```diff
@@ -45,7 +45,7 @@ public GameResultQueries(IUnitOfWork unitOfWork)
         }
 
         #endregion
-
+
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs]: What is the purpose of this change?

File: src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/TournamentsController.cs [hunk 3]
```diff
@@ -591,7 +591,7 @@ public JsonResult ApplyForTournament(GroupTeamViewModel groupTeam)
                     _requestService.Create(tournamentRequest);
                     result = Json(ViewModelResources.SuccessRequest);
                 }
-            }
+                }
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/TournamentsController.cs]: And this one?

File: tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs [hunk 4]
```diff
@@ -1590,6 +1591,35 @@ private List<SelectListItem> CreateTestTeamsList()
...
+                    Name="TeamNameB"
+                },
+                new TeamNameViewModel()
+                {
+                    DivisionName="DivisionNameA",
+                    Id=3,
+                    Name="TeamNameC"
+                }
+            };
+            expectedTeams.TournamentId = 1;
```
Comment (sdiachen) [inline tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs]: I believe there should be class level constant for tournament id

File: tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs [hunk 5]
```diff
@@ -180,21 +180,22 @@ public void Archived_ArchivedTournamentsExist_OnlyArchivedTournamentsReturned()
         public void ManageTournamentTeams_TournamentTeamsExist_TeamsInCurrentTournamentAreReturned()
         {
             // Arrange
+            var expextedTeamsList = CreateExpectedTeamsList();
```
Comment (MikolaB) [inline tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs]: expeCted

File: tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs [hunk 6]
```diff
@@ -1590,6 +1591,37 @@ private List<SelectListItem> CreateTestTeamsList()
...
+                    Id=3,
+                    Name="TeamNameC"
+                }
+            },
+                TournamentId = TEST_DIVISION_ID
+            };
+
+            return expectedTeams;
+        }
+
```
Comment (MikolaB) [inline tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs]: Please format code.

File: tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs [hunk 7]
```diff
@@ -1590,6 +1591,37 @@ private List<SelectListItem> CreateTestTeamsList()
             };
         }
 
+        private TournamentTeamsListViewModel CreateExpectedTeamsList()
+        {
+            TournamentTeamsListViewModel expectedTeams = new TournamentTeamsListViewModel
```
Comment (MikolaB) [inline tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs]: You can use var here.

File: tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs (hunk 6 above)
Comment (oleggerus) [reply tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs]: I don't know exactly how should it look like?

Comment (MikolaB) [reply tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs]: http://visualstudioshortcuts.com/2017/
Search for 'format document'

Comment (oleggerus) [reply tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs]: What exactly should change?

Comment (MikolaB) [reply tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs]: Did you try "Ctrl+K, Ctrl+D" keyboard shortcut in this file?

Comment (oleggerus) [reply tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs]: Yes, here is a new version.

File: src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs [hunk 8]
```diff
@@ -44,8 +44,7 @@ public GameResultQueries(IUnitOfWork unitOfWork)
             _dalGroups = _unitOfWork.Context.Groups;
         }
 
-        #endregion
-
+        #endregion
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs]: Please revert this change

File: tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs (hunk 6 above)
Comment (sdiachen) [reply tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs]: @MikolaB on his laptop Format Document command is not working 😮

File: src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs [hunk 9]
```diff
@@ -45,7 +45,7 @@ public GameResultQueries(IUnitOfWork unitOfWork)
         }
 
         #endregion
-
+
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs]: There should be no changes.

Comment (oleggerus) [reply src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs]: Ad I understand, row 48 should be empty and without spaces?

Comment (sdiachen) [reply src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs]: This file is not required to be changed to fix your task

Comment (oleggerus) [reply src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs]: I know, so how can I turn it to the previous state? Cause now it this file looks similar like in the master branch.

Comment (oleggerus) [reply src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs]: Maybe it will be easier to delete this branch and create the new one with only needed changes?

Comment (sdiachen) [reply src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs]: I highly doubt that :) Check you editor settings in VS
